bundles/net.tourbook.export/src/net/tourbook/export/DialogExportTour.java

```diff
@@ -1443,20 +1469,15 @@ private void setError(final String message) {
    private void setFileName() {
 
       // search for the first tour
-      TourData minTourData = null;
+      TourData minTourData = _tourDataList.get(0);
       final long minTourMillis = 0;
 
       for (final TourData tourData : _tourDataList) {
 
-         if (minTourData == null) {
-            minTourData = tourData;
-         } else {
-
-            final long tourMillis = tourData.getTourStartTime().toInstant().toEpochMilli();
+         final long tourMillis = tourData.getTourStartTime().toInstant().toEpochMilli();
 
-            if (tourMillis < minTourMillis) {
-               minTourData = tourData;
-            }
+         if (tourMillis < minTourMillis) {
+            minTourData = tourData;
          }
       }
```

Comment:
`minTourMillis` was/is wrong before and after this fix.
`minTourMillis` must be set when/from the value of `minTourData` is set

Reply:
Good to know. Fixed